src/data/recipes.js

```diff
@@ -56,6 +56,20 @@ export const butter = itemify({
   recipeType: recipeType.KITCHEN,
 })
 
+/**
+ * @property farmhand.module:recipes.sunButter
+ * @type {farmhand.recipe}
+ */
+export const sunButter = itemify({
+  id: 'sun-butter',
+  name: 'SunButter',
+  ingredients: {
+    [items.sunflower.id]: 8,
```

Comment:
What do you think about increasing this to roughly 25-50 or so? I imagine that sunflower seed butter requires a lot of seeds in real life (though I couldn't find any hard figures), and it would be good to make this late-game content more challenging as well.

Reply:
>  it would be good to make this late-game content more challenging as well.

i definitely agree with this! i don't think sunbutter requires quite as extreme of an amount as the oils do since (i believe) the whole kernel becomes the butter. and apparently one sunflower can produce up to 2k of them!? i think 25ish fits the game well though.

Reply:
25 seems like a good number to go with!